index.js

```diff
@@ -20,6 +20,7 @@ function safeCheck(file) {
 }
 
 module.exports = async (patterns, {force, dryRun, ...options} = {}) => {
+	options = {expandDirectories: false, onlyFiles: false, ...options};
```

Comment:
Most likely you want to disable the `followSymbolicLinks` option here (`true` by default). Otherwise, a `fast-glob` will look inside the directory referenced by the symbolic link.